### PR TITLE
Added support on ACE console for setting per character admin rights and character renaming

### PR DESCRIPTION
**Pull request by LtRipley36706**
Also included in this PR is a fix for the propertybools (IsAdmin, etc) not always being set from database when a character logs in.

### CODE REVIEW

**Comment by LtRipley36706** [comment]
Ignore this PR, will close shortly.

File: Source/ACE/Database/CharacterDatabase.cs [hunk 1]
```diff
@@ -117,6 +123,90 @@ public uint GetMaxId()
...
+            {
+                uint lowGuid = result.Result.Read<uint>(0, "guid");
+
+                if (boolId > 0)
+                    ExecutePreparedStatement(CharacterPreparedStatement.CharacterPropertiesBoolInsert, lowGuid, boolId, 1);
+                else
+                {
+                    ExecutePreparedStatement(CharacterPreparedStatement.CharacterPropertiesBoolInsert, lowGuid, (uint)PropertyBool.IsAdvocate, 0);
+                    ExecutePreparedStatement(CharacterPreparedStatement.CharacterPropertiesBoolInsert, lowGuid, (uint)PropertyBool.IsSentinel, 0);
+                    //ExecutePreparedStatement(CharacterPreparedStatement.CharacterPropertiesBoolInsert, lowGuid, (uint)PropertyBool.IsEnvoy, 0);
```
Comment (NathanielWaggoner) [inline Source/ACE/Database/CharacterDatabase.cs]: clean out commented code?  Do envoys not exist?

Comment (LtRipley36706) [inline Source/ACE/Database/CharacterDatabase.cs]: I had forgot at the time what IsEnvoy really was, which was IsPSR.

In anycase, this PR is no good anyway. Most of it has already been rolled into master via another PR and is mostly here just to remind me to re-do character rename in another PR soon.

**Comment by LtRipley36706** [comment]
PR #116 addresses the last change this PR was working on.
